index.html

```diff
@@ -52,133 +55,178 @@ <h1 class="intro">I'm Hammas Glad to see you!</h1>
+                        </ul>
+                       
+                    </div>
                 </div>
-                <button class="w1btn3">Get my resume</button>
-                <ul class="abm-skills-ul">
-                    <li class="abm-skills write">Languages</li>
-                    <li class="abm-skills-arrow"><img src="./images/arrow.png" alt="arrow-down"></li>
-                    <li class="abm-skills-img"><img class="abm-skills-img2" src="./images/js.png" alt="javaScript">
-                        <h3 class="abm-skills-img3">JavaScript</h3>
-                    </li>
-                    <li class="abm-skills-img"><img class="abm-skills-img2" src="./images/html.png" alt="html">
-                        <h3 class="abm-skills-img3">HTML</h3>
-                    </li>
-                    <li class="abm-skills-img"><img class="abm-skills-img2" src="./images/css.png" alt="css">
-                        <h3 class="abm-skills-img3">CSS</h3>
-                    </li>
-                    <li class="abm-skills write">Frameworks</li>
-                    <li class="abm-skills-arrow"><img src="./images/arrow2.png" alt="arrow-up"></li>
-                    <li class="abm-skills write">Skills</li>
-                    <li class="abm-skills-arrow"><img src="./images/arrow2.png" alt="arrow-up"></li>
-                </ul>
             </div>
         </div>
     </section>
-    <footer class="contact-form">
+    <footer class="contact-back">
+        <div class="contact-form">
         <form class="contact-info" action="https://formspree.io/f/mwkyenjw" method="POST">
             <h2>Contact me</h2>
-            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project
+                that needs coding. I’d love to help with it </p>
             <div class="input-container">
-                <input type="text" name="Namee" class="contact-input" maxlength="30" required placeholder="Write your name">
+                <input type="text" name="Name" class="contact-input" maxlength="30" required
+                    placeholder="Write your name">
                 <input type="email" name="_replyTo" class="contact-input" placeholder="Write your email" required>
-                <textarea name="message" class="contact-textarea"  cols="30" rows="10" required maxlength="500" placeholder="Write your message here"></textarea>
+                <textarea name="message" class="contact-textarea" cols="30" rows="10" required maxlength="500"
+                    placeholder="Write your message here"></textarea>
                 <input type="submit" class="contact-button" value="Get It Touch">
             </div>
             <img class="home-ind" src="./images/home-indicator.png" alt="home_indicator">
         </form>
+        </div>
```

Comment:
- [ ] (Optional) It would be nice if you considered using a `background-image` in CSS rather than an actual image. This is because the image in this section is part of the design and not an actual element or detail to the portfolio.